baseledger/x/bridge/keeper/keeper.go

```diff
@@ -71,6 +71,37 @@ func NewKeeper(
 	return k
 }
 
+/////////////////////////////
+//       PARAMETERS        //
+/////////////////////////////
+
+// GetWorktokenEurPrice returns the EUR price of a single worktoken. This is the price
+// used during calculation of how many worktokens to send to a UBT depositor in the brigde contract.
+//
+// This parameter can be changed through a governance param change proposal.
+func (k Keeper) GetWorktokenEurPrice(ctx sdk.Context) string {
```

Comment:
extreme nitpick but i was seeing this syntax as well, no need to change if  you prefer like this
func (k Keeper) GetWorktokenEurPrice(ctx sdk.Context) (a string) {
	k.paramstore.Get(ctx, types.ParamsStoreKeyWorktokenEurPrice, &a)
	return
}
EDIT: file below is actually using this syntax as well, just saw it immediately below :D

Reply:
i ll apply it